lib/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "tsc && vite build",
     "build:check": "if [ ! -d dist ] || [ $(find src -type f -newer dist) ] ; then yarn build ; fi",
-    "build:watch": "yarn build --watch",
+    "build:dev": "vite build --watch --logLevel warn --mode development",
```

Comment:
yarn build = tsc && vite build, y'a une raison en particulier de le changer?

Reply:
Yes, in development, I do not want to break build command if there are typescript errors.